src/routes/index.ts

```diff
@@ -12,6 +12,9 @@ import { validateSchema, UserSchema } from '../validations/user.validation'
 
 
 
+
+
+
 const router = express.Router();
```

Comment:
Please index.ts should not be like this, here are suggestions:
`
export * from "./auth.route";
export * from "./profile.route"`